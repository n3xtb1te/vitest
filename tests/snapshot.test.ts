import { expect, it } from 'vitest';

it('should match snapshot', () => {
    expect({ a: 1, b: 2 }).toMatchSnapshot();
});

it('should match snapshot v2', () => {
    const result = 'hello'.toUpperCase();
    expect(result).toMatchSnapshot();
});

it('should match inline snapshot', () => {
    expect({ a: 1, b: 2 }).toMatchInlineSnapshot(`
      {
        "a": 1,
        "b": 2,
      }
    `);
});