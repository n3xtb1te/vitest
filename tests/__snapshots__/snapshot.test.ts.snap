// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
{
  "a": 1,
  "b": 2,
}
`;

exports[`should match snapshot v2 1`] = `"HELLO"`;
